### DISCUSSION TITLE
Min Max Time for Datetime picker

### DISCUSSION THREAD

**Discussion by 7met** [wireui/wireui#323] [General]
On the datetime picker you can set a min and max date, is there a way to make the selectable time in a range. I want to be able to make it so you can only select time between 8am to 6pm.

**Comment by ph7jack** [comment]
@7met  At the moment is possible to scope only the initial time and the final time.
Is really nice to have a feature to scope the times on all days, I'll implement it for you.
For now, you can use the laravel validations to prevent times out of scope.
I'll release it tomorrow or the next Sunday

**Reply by 7met** [reply]
Thanks @PH7-Jack , that is amazing.

**Reply by ph7jack** [reply]
@7met I was sick in the previous week, I'm back now. Sorry for the delay. I'm finishing the implementation

**Reply by 7met** [reply]
No problem at all, we all have lives :)

I hope you are feeling better.

**Reply by ph7jack** [reply]
@7met Thanks for waiting, I've released the new feature now. You can see it here https://livewire-wireui.com/docs/datetime-picker#min-max-times

**Reply by 7met** [reply]
This is awesome! Thank you for putting this in. Amazing as always.
